src/app.template.json

```diff
@@ -16,6 +16,6 @@
     },
     "runtime": {
         "arguments": "--enable-aggressive-domstorage-flushing",
-        "version": "9.61.33.32"
+        "version": "beta"
```

Comment:
Do we want to tie this to beta or a specific version? What is this used for again?